src/components/NavigationBox.tsx

```diff
@@ -44,7 +44,7 @@ interface ItemProps {
 }
 
 interface Props {
-  heading?: string;
+  heading?: ReactNode;
```

Comment:
`heading: ReactNode | string` ?

For vi sender vell inn for det meste strenger uansett?